feat: add README and prepare release (#21)
shopware/paypal-sdk: README.md

<p align="center">
    <img src=".github/paypal-php-sdk.svg" width="200" alt="PayPal Logo" />
</p>

<p align="center">
    <h1 align="center">PayPal SDK for PHP</h1>
</p>

A Paypal PHP SDK that allows integration of PayPal services into PHP applications.
This SDK provides a simple and easy-to-use interface for working with PayPal's REST APIs, making it easier to accept payments, handle PayPal-related tasks and errors.

> [!WARNING]
> Currently this SDK is an extraction of the implementation within Shopware's [PayPal plugin](https://github.com/shopware/SwagPayPal).
> Only use cases and API responses used within the plugin are covered.

## Quick start
### Installation
You can install the SDK via Composer. Run the following command in your terminal:

```bash
composer require shopware/paypal-sdk
```

In addition, a PSR client implementation is needed. We recommend [Guzzle HTTP](https://github.com/guzzle/guzzle) or Symfony's [HttpClient](https://symfony.com/doc/current/http_client.html).                                               
```bash

> composer require guzzlehttp/psr7 php-http/guzzle7-adapter

Reviewer: Add these to the suggest section of the composer.json

https://getcomposer.org/doc/04-schema.md#suggest

<p align="center">
    <img src="https://www.paypalobjects.com/webstatic/de_DE/i/de-pp-logo-200px.png" width="150" alt="PayPal Logo" />
</p>
<p align="center">
    <b>+</b>
</p>
<p align="center">
    <img src="https://www.php.net/images/logos/new-php-logo.png" width="80" alt="PHP Logo" />

> </p>

Reviewer 2: ![image](https://github.com/user-attachments/assets/86528f8b-d19f-419a-9ba8-3a87349f4441)

![image](https://github.com/user-attachments/assets/180683ef-4e6d-4358-a26d-429ee43cc071)
thanks chatGPT

<p align="center">
    <h1 align="center">PayPal SDK for PHP</h1>
</p>

> A PayPal SDK for PHP that allows you to integrate PayPal services into your PHP applications.

Reviewer 2: ```suggestion
A Paypal PHP SDK that allows integration of PayPal services into PHP applications
```

This SDK provides a simple and easy-to-use interface for working with PayPal's REST APIs, making it easier to accept payments, handle PayPal-related tasks and errors.

> [!WARNING]
> Currently this SDK is an extraction of the implementation within Shopware's [PayPal plugin](https://github.com/shopware/SwagPayPal).
> Only use cases and API responses used within the plugin are covered.

## Quick start
### Installation
You can install the SDK via Composer. Run the following command in your terminal:

```bash
composer require shopware/paypal-sdk
```

> Additionally, a PSR client implementation is needed. Install [Guzzle HTTP](https://github.com/guzzle/guzzle) for example:

Reviewer 2: ```suggestion
In addition, a PSR client implementation is needed. We recommend [Guzzle HTTP](https://github.com/guzzle/guzzle):
```

```bash
composer require guzzlehttp/psr7 php-http/guzzle7-adapter
```

The [php-http/discovery](https://github.com/php-http/discovery) package is used for this library to discover the PSR client implementation automatically.
Have a look at [their README](https://github.com/php-http/discovery?tab=readme-ov-file#usage-as-a-library-user) to set the client implementation manually.

> Alternatively provide a gateway with a proper client.

Reviewer 2: ```suggestion
Alternatively you can provide a gateway with a concrete client of your choice.
```

<p align="center">
    <img src=".github/paypal-php-sdk.svg" width="200" alt="PayPal Logo" />
</p>

<p align="center">
    <h1 align="center">PayPal SDK for PHP</h1>
</p>

> A Paypal PHP SDK that allows integration of PayPal services into PHP applications.

Reviewer 3: PayPal!
